Created Output Log File: /workdir/tasks_run/data/logs/calculator_script_20240911_14h57m07s.txtdicom dir using: /workdir/tasks_run/data/sambashare/test_dir
Starting Block1 ... 
========================================
Starting Block1, Trial 1... 
========================================
Waiting For New File ...
New File Found In Dir...
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/73413465622653788565.dcm
Error:
Cannot Find Nifti Image After dcm2niix
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 32, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 112, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"] = file_handler.dicom_to_nifti(dicom_file=dicom_path, trial=trial)
  File "/workdir/tasks_run/scripts/file_handler.py", line 69, in dicom_to_nifti
    raise Exception("Cannot Find Nifti Image After dcm2niix")
Exception: Cannot Find Nifti Image After dcm2niix

Retries left: 3
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/73413465622653788565.dcm
Error:
Cannot Find Nifti Image After dcm2niix
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 32, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 112, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"] = file_handler.dicom_to_nifti(dicom_file=dicom_path, trial=trial)
  File "/workdir/tasks_run/scripts/file_handler.py", line 69, in dicom_to_nifti
    raise Exception("Cannot Find Nifti Image After dcm2niix")
Exception: Cannot Find Nifti Image After dcm2niix

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Retries left: 2
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/34696466688789809923.dcm
nii_img_dtpye: int16
Mean ROI Activation: 432.0824890136719
Inter-Trial Calculations Successful.
Ending Block1, Trial 1... 
========================================
Starting Block1, Trial 2... 
========================================
Waiting For New File ...
New File Found In Dir...
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/84048905510810050975.dcm
nii_img_dtpye: int16
Mean ROI Activation: 431.45361328125
Inter-Trial Calculations Successful.
Ending Block1, Trial 2... 
========================================
Starting Block1, Trial 3... 
========================================
Waiting For New File ...
New File Found In Dir...
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/64398245227446583796.dcm
Error:
Cannot Find Nifti Image After dcm2niix
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 32, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 112, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"] = file_handler.dicom_to_nifti(dicom_file=dicom_path, trial=trial)
  File "/workdir/tasks_run/scripts/file_handler.py", line 69, in dicom_to_nifti
    raise Exception("Cannot Find Nifti Image After dcm2niix")
Exception: Cannot Find Nifti Image After dcm2niix

Retries left: 3
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/64398245227446583796.dcm
Error:
Cannot Find Nifti Image After dcm2niix
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 32, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 112, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"] = file_handler.dicom_to_nifti(dicom_file=dicom_path, trial=trial)
  File "/workdir/tasks_run/scripts/file_handler.py", line 69, in dicom_to_nifti
    raise Exception("Cannot Find Nifti Image After dcm2niix")
Exception: Cannot Find Nifti Image After dcm2niix

Getting New Dicom Instead...
Waiting For New File ...
---- Keyboard Interrupt Detected ----
What Would You Like to Do?
(1) Continue With The Block
(2) End The Session
(3) Start New Block
Ok, Let's End the Session...
Nifti Outdir Cleared
Ending Session Due to: keyboard interrupt
Session Data:
Running Main Calculation Script ... 
